src/helpers/helpersAPI.js

```diff
@@ -0,0 +1,20 @@
+const getLocationsFromResponseAPI = (response = null) => {
+  if (response === null) return [];
+  if (typeof response.data === 'undefined') return [];
+  if (typeof response.data.query === 'undefined') return [];
```

Comment:
I like these 3 lines as short-circuits for when data isn’t available. As long as this works, let’s keep it as-is, but we might want to put a story on the board to refactor this function because I suspect there’s a shorter, more clear way to do the work of this function with optional chaining.

Reply:
Thanks for creating this helper function!